app/assets/javascripts/components/SlotPicker.es6

```diff
@@ -0,0 +1,121 @@
+(function() {
+  'use strict';
+
+  class SlotPicker extends PWPG.BaseComponent {
+    constructor($component) {
+      super($component);
+
+      this.$months = this.$component.find('.js-slot-picker-month');
+      this.$form = this.$component.parents('.js-day-picker-form');
```

Comment:
Thinking about this a little more, this could be a little fragile... it means that the `SlotPicker` wouldn't work in isolation, which I guess is the idea behind the component pattern.

I guess a way around this is to make the form (or highest parent) the component container. 

What do you reckon?